scripts/sts/create-rhoam-policy.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
This have been moved to https://github.com/integr8ly/delorean/pull/254 also but not sure should we keep this copy to aid local development :thinking:

Reply:
I think it's better to have a single source of truth as the two scripts may change over time